ml_perf/reference_implementation.py

```diff
@@ -414,7 +423,12 @@ def rl_loop():
                                 '*')
 
     # Calculate the window size from which we'll select training chunks.
-    window = 1 + state.iter_num
+    if not FLAGS.check_point:
+      window = 1 + state.iter_num
+    else:
+      window = (FLAGS.slow_window_size
```

Comment:
I don't really understand this logic. To compute the window size correctly, you have to take into account the number of training chunks that were in the checkpoint.

It might be easiest to move the logic that computes the window size directly into `get_golden_chunk_records` and calculate it from the number of files matched by the `*.zz` glob instead of `state.iter_num`.

Reply:
This logic is to make sure window size is max_windows_size when training started, and yes, this logic needs improve.
I'll modify the code to be more flexible according to suggestion, and think over this part.

Reply:
the "slow window" flag was only needed when we were starting from random noise; its purpose is to get the initial random games out of the window sooner, because we had thought that the initial way out of the randomness was right about when the window would fill up.  That's not actually the case, apparently, so we can actually get rid of the whole slow-window flag in the reference implementation.

Reply:
The logic is improved.  The new logic also support chunk number between 1 and max_window_size, so a premature checkpoint can be replicated and window size would grow to max_window_size.

I think the original train from scratch mode should be kept as optional in reference implementation, so people can actually train from scratch and know the implementation is following the spirit of reinforcement learning.